umm-spec-lib/resources/json-schemas/variable/umm/v1.9/umm-var-search-results-json-schema.json

```diff
@@ -0,0 +1,286 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "ItemType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Represents a single item found in search results. It contains some metadata about the item found along with the UMM representing the item. umm won't be present if the item represents a tombstone.",
+      "properties": {
+        "meta": {
```

Comment:
I'm in favor of single property objects being on one line if you want to make it more compact, unless software did this formatting for you, then do what ever the software did.

Reply:
I didn't change this file. I just brought it forward. Looking at the code base, this was in the initial release 6 years ago.  I will leave it as is.